enhancements/ingress/optional-ingress.md

```diff
@@ -0,0 +1,274 @@
+---
+title: optional-ingress
+authors:
+  - "@alebedev87"
+reviewers:
+  - "@Miciah"
+  - "@bparees"
+  - "@cewong"
+  - "@wking"
+approvers:
+  - "@Miciah"
+  - "@bparees"
+api-approvers:
+  - N/A
+creation-date: 2023-05-30
+last-updated: 2023-05-30
+tracking-link:
+  - https://issues.redhat.com/browse/NE-1129
+see-also:
+  - "/enhancements/installer/component-selection.md"
+  - "/enhancements/ingress/aws-load-balancer-operator.md"
+replaces:
+superseded-by:
+---
+
+# Make Ingress Operator optional
+
+## Summary
+
+This enhancement describes the implementation of a new capability to enable/disable the default ingress of the OpenShift cluster.
+The new capability (from here on called the ingress capability) includes [the cluster ingress operator](https://github.com/openshift/cluster-ingress-operator) as well as the custom resources managed by the operator.
+This enhancement focuses on the requirements for [RFE-3395](https://issues.redhat.com/browse/RFE-3395) which targets ROSA, a managed OpenShift service based on HyperShift. The implementation of the ingress capability on the standalone OpenShift is out of the scope of this enhancement.
+
+## Motivation
+
+Users of the ROSA managed OpenShift service would like to use [AWS load balancers](https://github.com/openshift/enhancements/blob/master/enhancements/ingress/aws-load-balancer-operator.md) as a way to accept the ingress traffic into the cluster.
+The cluster ingress operator becomes unnecessary and can be disabled to reduce the resource consumption.
+
+### Goals
+- Implement the new ingress capability.
+- Add the ingress capability to the ingress operator's payload.
+- Forbid the usage of the ingress capability for the standalone OpenShift.
+- Make the dependent cluster operators tolerate the absence of the ingress operator.
+
+### Non-Goals
+- Use a different way for enabling/disabling the ingress other than via [the cluster capability](https://github.com/openshift/enhancements/blob/master/enhancements/installer/component-selection.md).
```

Comment:
nit: you don't need to pin the org/repo/branch here; instead you can use a relative-to-this-file link like `../installer/component-selection.md` or a relative-to-the-repo-root link like `/enhancements/installer/component-selection.md`.

Reply:
Done.